templates/angular-package/_env

```diff
@@ -0,0 +1 @@
+TEST="test value with Rafa"
```

Comment:
Does this file need to be commited?

Reply:
hahahaha, thanks i will remove the value